pkg/sources/github/github.go

```diff
@@ -679,56 +748,42 @@ func (s *Source) getReposByOrg(ctx context.Context, org string) ([]string, error
 			continue
 		}
 		if err != nil {
-			return nil, fmt.Errorf("could not list repos for org %s: %w", org, err)
+			return fmt.Errorf("could not list repos for org %s: %w", org, err)
 		}
 		if len(someRepos) == 0 || res == nil {
 			break
 		}
 
 		logger.V(2).Info("Listed repos", "page", opts.Page, "last_page", res.LastPage)
 		for _, r := range someRepos {
-			if s.ignoreRepo(r.GetFullName()) {
-				continue
-			}
-			if !s.includeRepo(r.GetFullName()) {
-				continue
-			}
-
-			numRepos++
 			if r.GetFork() {
-				numForks++
 				if !s.conn.IncludeForks {
 					continue
 				}
+				numForks++
+			}
+			repo, err := s.normalizeRepo(r.GetCloneURL())
+			if err != nil {
```

Comment:
what could cause repo url normalization failures? are we ok with skipping repos when it happens?